components/builder-worker/src/runner/mod.rs

```diff
@@ -671,7 +706,7 @@ impl RunnerMgr {
         rz.send(()).unwrap();
 
         let mut srv_msg = false;
-        let (tx, rx): (_, mpsc::Receiver<Job>) = mpsc::channel();
+        let (tx, mut rx): (_, async_mpsc::Receiver<Job>) = async_mpsc::channel(1);
```

Comment:
We should change this to use `async_mpsc::unbounded_channel`

Reply:
Addressed in a33367f0febdd930dfb5cba3892af203fc5b3a51